### DISCUSSION TITLE
Cant install any model

### DISCUSSION THREAD

**Discussion by Alfabis** [carson-katri/dream-textures#624] [General]
Whenever i try to install an model i either get stuck right at the beginning on 0% or get stuck at 33% at tokenizer/merges.txt. Any solution to that?

**Comment by TomHardy02** [comment]
Let me know how you go, I am having the same issue.

**Comment by MirlanAskingQuestions** [comment]
Greetings from 2024. 
Let me know how you go, I am having the same issue.

**Comment by brandoirl** [comment]
Is there any fix for this yet? I also have the same problem
